Tuisky does not with Cargo on Windows 10 Powershell

Hi
I tried to install tuisky with rustc 1.82. But, I got this error message: 

error[E0433]: failed to resolve: could not find `unix` in `os`
  --> C:\Users\dtant\.cargo\registry\src\index.crates.io-6f17d22bba15001f\tuisky-0.1.4\src\components\modals\embed_images.rs:12:14
   |
12 | use std::os::unix::fs::MetadataExt;
   |              ^^^^ could not find `unix` in `os`
   |
note: found an item that was configured out
  --> /rustc/f6e511eec7342f59a25f7c0534f1dbea00d01b14\library\std\src\os\mod.rs:27:9
note: the item is gated here
  --> /rustc/f6e511eec7342f59a25f7c0534f1dbea00d01b14\library\std\src\os\mod.rs:19:1
note: found an item that was configured out
  --> /rustc/f6e511eec7342f59a25f7c0534f1dbea00d01b14\library\std\src\os\mod.rs:65:9
note: the item is gated here
  --> /rustc/f6e511eec7342f59a25f7c0534f1dbea00d01b14\library\std\src\os\mod.rs:64:1

error[E0599]: no method named `size` found for struct `std::fs::Metadata` in the current scope
  --> C:\Users\dtant\.cargo\registry\src\index.crates.io-6f17d22bba15001f\tuisky-0.1.4\src\components\modals\embed_images.rs:93:33
   |
93 |                     && metadata.size() <= 1_000_000
   |                                 ^^^^ method not found in `Metadata`

Some errors have detailed explanations: E0433, E0599.
For more information about an error, try `rustc --explain E0433`.
error: could not compile `tuisky` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: failed to compile `tuisky v0.1.4`, intermediate artifacts can be found at `C:\Users\dtant\AppData\Local\Temp\cargo-installJLwa4V`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
What do you think?
![image](https://github.com/user-attachments/assets/4b9d8c7e-7d9d-4c85-9504-21e7e0d24f6b)

--- a/src/components/modals/embed_images.rs
+++ b/src/components/modals/embed_images.rs
@@ -9,7 +9,6 @@ use ratatui::style::{Color, Style, Stylize};
 use ratatui::text::Line;
 use ratatui::widgets::{Block, Clear};
 use ratatui::Frame;
-use std::os::unix::fs::MetadataExt;
 use std::path::PathBuf;
 use tui_textarea::TextArea;
 
@@ -90,7 +89,7 @@ impl EmbedImagesModalComponent {
             let path = PathBuf::from(self.image.path.lines().join(""));
             self.state = if let Ok(metadata) = path.metadata() {
                 if metadata.is_file()
-                    && metadata.size() <= 1_000_000
+                    && metadata.len() <= 1_000_000
                     && ImageReader::open(path)
                         .ok()
                         .and_then(|reader| reader.decode().ok())

